system/src/Grav/Common/Language/Language.php

```diff
@@ -472,12 +472,13 @@ public function getTranslation($lang, $key, $array_support = false)
     public function getBrowserLanguages($accept_langs = [])
     {
         if (empty($this->http_accept_language)) {
-            if (empty($accept_langs) && isset($_SERVER['HTTP_ACCEPT_LANGUAGE'])) {
-                $accept_langs = $_SERVER['HTTP_ACCEPT_LANGUAGE'];
-            } else {
+
+            if (empty($accept_langs) == false && isset($_SERVER['HTTP_ACCEPT_LANGUAGE']) == false) {
```

Comment:
@Quy There are still a couple of issues with logic that @mahagr has pointed out.  We can't merge this until those are sorted.  Cheers.